cmd/verifier/main.go

```diff
@@ -27,7 +27,7 @@ import (
 	"time"
 
 	"github.com/sigstore/rekor-monitor/pkg/rekor"
-	file "github.com/sigstore/rekor-monitor/pkg/util"
+	file "github.com/sigstore/rekor-monitor/pkg/util/file"
```

Comment:
nit, you can remove `file` from the beginning of this declaration as this package is now named `file`